problem-list.md

```diff
@@ -34,8 +34,8 @@
 |[Remove Trailing Zeros From a String](https://leetcode.com/problems/remove-trailing-zeros-from-a-string/)|_TODO: add_|✅||
 |[Rings and Rods](https://leetcode.com/problems/rings-and-rods/)|_TODO: add_|✅||
 |[Kids With the Greatest Number of Candies](https://leetcode.com/problems/kids-with-the-greatest-number-of-candies/)|_TODO: add_|✅✅||
-|[Separate the Digits in an Array](https://leetcode.com/problems/separate-the-digits-in-an-array/)|_TODO: add_|✅✅||
-|[Ransom Note](https://leetcode.com/problems/ransom-note/)|_TODO: add_|✅||
+|[Separate the Digits in an Array](https://leetcode.com/problems/separate-the-digits-in-an-array/)|_TODO: add_|✅✅|✅|
+|[Ransom Note](https://leetcode.com/problems/ransom-note/)|_TODO: add_|✅✅|✅|
```

Comment:
I don't remember August Cohort doing Ransom Note

Reply:
I'm counting it for the day that the September cohort did since I asked the summer cohort to review the September cohort's code.

Reply:
That makes sense.

Reply:
Want to review the PR? 🙂